homework_05/15_martin_dinev/decrypt.py

```diff
@@ -0,0 +1,33 @@
+from posixpath import split
+from nltk.corpus import wordnet
+
+key = 0
+str = input()
+
+for i in range(0, -25, -1):
+    for word in str.split():
+        correctWord = ""
```

Comment:
Doesn't comply the snake case word syntax of python. This can be seen on several lines